packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1387,6 +1388,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     public flush(): void {
+        assert(
+            this._orderSequentiallyCalls === 0,
+            0x234 /* "Cannot call `flush()` from `orderSequentially`'s callback" */);
```

Comment:
This assert code should be auto-generated right?

Reply:
it was auto-generated